tests/unit/session-stores/adaptive-test.js

```diff
@@ -6,13 +6,18 @@ import Adaptive from 'ember-simple-auth/session-stores/adaptive';
 import itBehavesLikeAStore from './shared/store-behavior';
 import itBehavesLikeACookieStore from './shared/cookie-store-behavior';
 import FakeCookieService from '../../helpers/fake-cookie-service';
+import Configuration from 'ember-simple-auth/configuration';
 
 const { assign: emberAssign, merge, run } = Ember;
 const assign = emberAssign || merge;
 
 describe('AdaptiveStore', () => {
   let store;
 
+  beforeEach(function() {
+    Configuration.load({ baseURL: '/' });
```

Comment:
I see. I'd just set the `cookiePath` property on the created store instance instead ([here](https://github.com/simplabs/ember-simple-auth/blob/master/tests/unit/session-stores/shared/cookie-store-behavior.js#L98)) in these cases so the test is independent of the `Configuration`.